sharktank/tests/layers/mixture_of_experts_block_test.py

```diff
@@ -200,6 +203,106 @@ def testParityOfExpertPreGatherFfnAndDenseFfn(
         res_dense = moe_with_dense_ffn(input)
         torch.testing.assert_close(res_pre_gather, res_dense)
 
+    @parameterized.expand(
+        [
+            param(
+                dtype=torch.float32,
+                feature_dim=7,
+                expert_hidden_dim=3,
+                num_experts=12,
+                n_expert_groups=4,
+                n_limited_groups=2,
+                expert_used_count=2,
+                num_shared_experts=5,
+                shared_expert_hidden_dim=6,
+                batch_size=8,
+                sequence_length=9,
+                rms_epsilon=0.01,
+                moe_activation_fn=torch.nn.functional.silu,
+                score_experts_fn=torch.nn.functional.sigmoid,
+                normalize_experts=True,
+                add_residual=False,
+                route_scale=None,
+                tensor_parallelism_size=2,
+            ),
+        ]
+    )
```

Comment:
Did you mean to add multiple tests?

Reply:
I added one more test with another set of parameters and bf16 dtype.